app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -310,6 +310,14 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
             }
 
             webChromeClient = object : WebChromeClient() {
+                override fun onProgressChanged(view: WebView?, newProgress: Int) {
+                    if (newProgress == 100) {
+                        view?.loadUrl(
+                            "javascript:window.externalApp.getHeaderColor(" +
+                                    "document.getElementsByTagName('html')[0].computedStyleMap().get('--app-header-background-color')[0]);"
+                        )
+                    }
+                }
```

Comment:
Do we have to hook into the `onProgressChanged`?  Can we use the events on the external bus instead here?: https://github.com/home-assistant/android/pull/1431/files#diff-76bbedb2918a5db79c7070e9bb2e01d9241388745f60b0719c0d1f74123d89bdR439-R446

Reply:
Tried calling from the `connection-status` event on the external bus, but that one is too early. The `config/get` event was ok. But i don't know if this is the correct place to set the statusbar color? Maybe there are side effects? What do you think?